Source/Classes/NantesLabel.swift

```diff
@@ -58,14 +58,15 @@ public extension NSAttributedString.Key {
     public static let nantesLabelStrikeOut: NSAttributedString.Key = .init("NantesLabelStrikeOutAttribute")
 }
 
-open class NantesLabel: UILabel {
+@IBDesignable open class NantesLabel: UILabel {
     public struct Link: Equatable {
         public var attributes: [NSAttributedString.Key: Any]
         public var activeAttributes: [NSAttributedString.Key: Any]
         public var inactiveAttributes: [NSAttributedString.Key: Any]
         public var linkTappedBlock: NantesLabel.LinkTappedBlock?
         public var result: NSTextCheckingResult?
         public var text: String?
+   
```

Comment:
Would you mind removing this extra line?

Reply:
It looks like there's still some trailing whitespace here, would you mind removing it?

Reply:
On the PR I can look at different commits, my latest commit removes this whitespace

Reply:
Yeah the extra newline is gone, but it looks like your editor left some spaces behind on line 69. It's not a huge issue